### PR TITLE
Add teardown script

### PR DESCRIPTION
**Pull request by suyograo**

### CODE REVIEW

File: kafka_test_teardown.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# Setup Kafka and create test topics
+
+set -ex
+
+echo "Stopping Kafka broker"
+kafka/bin/kafka-server-stop.sh
+echo "Stopping zookeeper"
+kafka/bin/zookeeper-server-stop.sh
```
Comment (ph) [inline kafka_test_teardown.sh]: Could it be done in the rspec `after :all` block?

Comment (ph) [inline kafka_test_teardown.sh]: I am just wondering if can be done in the spec lifecycle we won't need to change anything in the travis.yml.

Comment (suyograo) [inline kafka_test_teardown.sh]: yeah, but this is only needed for the entire test suite.

**Comment by ph** [comment]
Nvermind LGTM

**Pull request merged**